src/gui/sdlmain.cpp

```diff
@@ -4263,7 +4270,7 @@ void Config_Add_SDL() {
 	        "fullscreen or when the mouse is captured in window mode.");
 
 	Pbool = sdl_sec->Add_bool("waitonerror", always, true);
-	Pbool->Set_help("Wait before closing the console if dosbox has an error.");
+	Pbool->Set_help("Wait before closing the console if DOSBox has an error.");
```

Comment:
Suggest `"Keep the close open if an error has occurred."`

Reply:
OK, changed.